.gitattributes

```diff
@@ -0,0 +1,69 @@
+# Handle line endings automatically for files detected as text
+# and leave all files detected as binary untouched.
+* text=auto
```

Comment:
`text=auto` doesn't do what you think it does.

Example: if a new PR is submitted from a Windows user that introduces new files those new files will be CRLF on the PR.

It would be better and more sane to just mark them all `lf` below (with the exception of `*.bat` / `*.cmd` like you have it)

We saw this quite a bit on Eclipse Jetty and switched away from `auto` intentionally.

Reply:
Please file an issue and we'll fix it in the following PRs.